Firestore/core/test/unit/testutil/bundle_builder.h

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef FIRESTORE_CORE_TEST_UNIT_TESTUTIL_BUNDLE_BUILDER_H_
+#define FIRESTORE_CORE_TEST_UNIT_TESTUTIL_BUNDLE_BUILDER_H_
+
+#include <array>
+#include <string>
+#include <vector>
+
+#include "Firestore/core/include/firebase/firestore/firestore_errors.h"
+#include "Firestore/core/src/util/status_fwd.h"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace firebase {
+namespace firestore {
+namespace testutil {
+
+namespace {
+
+std::array<std::string, 7> BundleTemplate() {
+  std::string metadata =
+      "{\"metadata\":{\"id\":\"test-bundle\",\"createTime\":{\"seconds\":1001,"
```

Comment:
It seems like [raw string literals](go/totw/64) could improve readability here (no need to escape the quotes) (assuming additional line breaks are okay).

Reply:
Done.